include/deal.II/grid/grid_tools_cache.h

```diff
@@ -150,6 +150,26 @@ namespace GridTools
     const Mapping<dim, spacedim> &
     get_mapping() const;
 
+
+    /**
+     * Its elements are pairs of bounding boxes denoting areas
+     * that cover all or parts of the local portion of a parallel
+     * triangulation, and an unsigned int representing the process
+     * or subdomain that owns these cells.
```

Comment:
Maybe add "While each box may only cover part of a process's locally owned part of the triangulation, the boxes associated with each process jointly cover the entire local portion."